src/aggregator/aggregator/aggregator.go

```diff
@@ -773,7 +777,11 @@ func (m *aggregatorAddMetricMetrics) ReportError(err error) {
 	case errWriteValueRateLimitExceeded:
```

Comment:
Does it make sense to make the `switch` just check a bool and have each of these be `case xerrors.Is(err, $target)`?